android/build.gradle

```diff
@@ -6,8 +6,8 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.0.1'
-        classpath 'com.google.gms:google-services:3.0.0'
+        classpath 'com.android.tools.build:gradle:3.1.2'
+        classpath 'com.google.gms:google-services:3.2.1'
```

Comment:
Is this required?

Reply:
![image](https://user-images.githubusercontent.com/13368542/40550898-4744da96-5ff0-11e8-9b98-95f03d243cf5.png)
![image](https://user-images.githubusercontent.com/13368542/40550916-5219b306-5ff0-11e8-98a5-32da7fd71b61.png)